lib/editor/tunnel.js

```diff
@@ -248,6 +248,11 @@ class EditorTunnel {
                     })
                 }
             })
+            clearInterval(this.pingInterval)
+            this.pingInterval = setInterval(() => {
+                debug('Sending WS editor tunnel ping')
+                socket.ping()
+            }, 15000)
```

Comment:
```suggestion
            }, 40000)
```

We see the timeouts at the 60s mark - so I think we can afford to reduce the ping frequency a bit here - will halve the traffic it generates otherwise :)

Reply:
They about as small as we can send, but good point. I was just too impatient to wait that long while testing....